content/themen/bildung.en.md

```diff
@@ -8,24 +8,24 @@ img_social: /okf/bildung-social-og.png
 color: "#382eff"
 ---
 
-Open education is an umbrella term for school-based and extracurricular initiatives that understand education to be participatory, open and relevant to current issues and trends. Education is promoted as a common good and focus is placed on self-directed and competence-oriented learning.
+Open education is an umbrella term for in-school and extracurricular initiatives that promote participatory education and adapt educational concepts to the digital era. These initiates see education as a common good and place special focus on self-directed and competence-based learning.
 
 <!--more-->
 
-Open education promotes:
+## Open education promotes:
 
-1. Digitale Mündigkeit (Digital Maturity)
-2. the development of new forms of social and civic engagement, e.g. through digital volunteering
-3. the ethical handling of information and communication technologies (see: [Hacker Ethics](https://www.ccc.de/en/hackerethics))
+1. Digital literacy
```

Comment:
It seems like you really want to use the term "Mündigkeit" instead of literacy. Is there a reason? If so, I would recommend explaining that and making it consistent across the site.
(FWIW, I think literacy captures the ideas of Mündigkeit!)

Reply:
literacy: "ability to read / write, knowledge of a particular subject, or a particular type of knowledge"
maturity: "ability to respond to the environment in an appropriate manner"
The OKF edu projects want to help young people to develop a critical and proactive relationship with technology, which does not necessarily coincide with / goes beyond literacy. This is why we chose to use maturity rather than literacy.

Reply:
There is a point to be made, however, for using literacy, as it seems to be commonly used to describe projects in the realm of digital / open education in the English-speaking world and maturity thus far seems to have been mainly used by managers and economic actors when describing the digital transformation of companies.